logs_sdk/lib/opentelemetry/sdk/logs/log_record_processor.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module SDK
+    module Logs
+      # Presently no-op LogRecordProcessor
+      class LogRecordProcessor
```

Comment:
No 'onEmit' method?

Reply:
The LoggerProvider specs don't refer to the `onEmit` method, so I thought I'd introduce that functionality in a separate PR. I'm open to adding it if you think it'd clarify things.

Reply:
@mwear - Sounds great. Updates made in: 26ef80d0fd93561f1283ca32f50460db760fd4cc